app/views/spree/admin/payments/source_views/_paypal_braintree.html.erb

```diff
@@ -1,10 +1,15 @@
 <fieldset data-hook="credit_card">
+  <legend align="center"><%= t('braintree', scope: 'solidus_paypal_braintree') %></legend>
```

Comment:
Why this `legend`? No other admin payment view has this legend and admins do not have to know about the provider of this payment method IMO. But I'm not considering this a blocker.